File: app/src/main/java/com/goals/ted/goals/DateFormatter.java
package com.goals.ted.goals;

import java.text.SimpleDateFormat;
import java.util.Calendar;

/**
 * Created by ted on 9/20/16.
 */
public class DateFormatter {

    public static String convertDateToDateFormat(Calendar date){
        String formattedDate = null;
        SimpleDateFormat format1 = new SimpleDateFormat("EEE, MMM dd yyyy");
        formattedDate = format1.format(date.getTime());
        return formattedDate;

    }
}

File: app/src/main/java/com/goals/ted/goals/DateFormatter.kt
package com.goals.ted.goals

import java.text.SimpleDateFormat
import java.util.Calendar

/**
 * Created by ted on 9/20/16.
 */
object DateFormatter {

    fun convertDateToDateFormat(date: Calendar?): String {
        var formattedDate: String? = null
        val format1 = SimpleDateFormat("EEE, MMM dd yyyy")
        formattedDate = format1.format(date?.time)
        return formattedDate

    }
}
